app/views/layouts/application.html.erb

```diff
@@ -10,10 +11,12 @@
   </head>
 
   <body>
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <%= image_tag 'go_local.png', class: "navbar-brand", style: 'width:80px' %>
-      <%= link_to 'Logout', logout_path unless !current_user %>
-    </nav>
+    <% if current_user %>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <%= image_tag 'go_local.png', class: "navbar-brand", style: 'width:80px' %>
+        <%= link_to 'Logout', logout_path unless !current_user %>
```

Comment:
We can get rid of the guard clause since the whole header now depends on having a current user

Reply:
Sure!